src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertyAccessorSymbolReferenceFinder.cs

```diff
@@ -34,14 +32,64 @@ protected override bool CanFind(IMethodSymbol symbol)
             return result;
         }
 
-        protected override Task<ImmutableArray<Document>> DetermineDocumentsToSearchAsync(IMethodSymbol symbol, Project project, IImmutableSet<Document> documents, CancellationToken cancellationToken)
+        protected override async Task<ImmutableArray<Document>> DetermineDocumentsToSearchAsync(IMethodSymbol symbol, Project project, IImmutableSet<Document> documents, SymbolFinderOptions options, CancellationToken cancellationToken)
         {
-            return FindDocumentsAsync(project, documents, cancellationToken, symbol.Name);
+            var explicitUsages = await FindDocumentsAsync(project, documents, cancellationToken, symbol.Name).ConfigureAwait(false);
+
+            var property = symbol.AssociatedSymbol as IPropertySymbol;
+            if (property == null || !options.IncludeImplicitAccessorUsages)
```

Comment:
what about events?

Reply:
Events will be done in `EventAccessorSymbolReferenceFinder` which does not exist yet. (Item 4 on TODO in PR body)